Sources/ArgumentParser/Parsable Types/ParsableArguments.swift

```diff
@@ -60,6 +60,28 @@ struct _WrappedParsableCommand<P: ParsableArguments>: ParsableCommand {
   @OptionGroup var options: P
 }
 
+/// A type that provides the `ParsableCommand` interface to a `ParsableArguments` type,
+/// and set the `shouldPromptForMissing` to true.
+struct _WrappedThrowableCommand<P: ParsableArguments>: ParsableCommand {
```

Comment:
Why are we using a new type here instead of modifying `_WrappedParsableCommand`? Is the old type still used anywhere?